### PR TITLE
Feat/AR 764

### PR DESCRIPTION
**Pull request by aaron-bcw**
## Description
Added several tests for the following: 
* eth_syncing
* net_listening
* net_version
* web3_clientVersion

* updated packages

> Note: 
Working on the sendRawTransaction in a seperate story

### CODE REVIEW

**Review by syed-bcw — APPROVED** [review]
👍👍👍

File: nodejs/json-rpc-relay/integration.tests/mirror.node/net.version.test.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        const configPayload = getPayload();
+
+        // Act
+        const { data } = await curly.post(arkhiaJsonRpcRelayTestnet, {
+            postFields: JSON.stringify(configPayload),
+            httpHeader: httpHeaderJson,
+        });
+
+        // Assert
+        expect(data?.result).toBeDefined();
```
Comment (daniel-bcw) [inline nodejs/json-rpc-relay/integration.tests/mirror.node/net.version.test.js:38]: Wonder , would we want also to define its 296 id?

File: nodejs/json-rpc-relay/integration.tests/mirror.node/eth.syncing.test.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+        // Act
+        const { data } = await curly.post(arkhiaJsonRpcRelayTestnet, {
+            postFields: JSON.stringify(configPayload),
+            httpHeader: httpHeaderJson,
+        });
+
+        // Assert
+        expect(data.result).toBeDefined();
+        expect(JSON.parse(data.result)).toBeFalsy()
```
Comment (daniel-bcw) [inline nodejs/json-rpc-relay/integration.tests/mirror.node/eth.syncing.test.js:39]: Why falsy here

File: nodejs/json-rpc-relay/integration.tests/mirror.node/net.listening.test.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        "method": "net_listening",
+        "params": [],
+        "id": 0
+      };
+    return data;
+}
+
+describe('[CURL] Net Listening', () => {
+
+    test('Should return false from Arhia Testnet', async () => {
```
Comment (daniel-bcw) [inline nodejs/json-rpc-relay/integration.tests/mirror.node/net.listening.test.js:27]: Should it return false

File: nodejs/json-rpc-relay/integration.tests/mirror.node/net.version.test.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+    test('Should get current chain id from Arhia Testnet', async () => {
+        // Arrange
+        const web3Provider = new Web3(arkhiaJsonRpcRelayTestnet);
+
+        // Act
+        const result = await web3Provider.eth.net.getId();
+
+        // Assert
+        expect(result).toBeDefined();
```
Comment (daniel-bcw) [inline nodejs/json-rpc-relay/integration.tests/mirror.node/net.version.test.js:52]: What is the id, should we check if the id is `296` for testnet network ?

File: nodejs/json-rpc-relay/integration.tests/mirror.node/web3.clientVersion.test.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        const configPayload = getPayload();
+
+        // Act
+        const { data } = await curly.post(arkhiaJsonRpcRelayTestnet, {
+            postFields: JSON.stringify(configPayload),
+            httpHeader: httpHeaderJson,
+        });
+
+        // Assert
+        expect(data?.result).toBeDefined();
```
Comment (daniel-bcw) [inline nodejs/json-rpc-relay/integration.tests/mirror.node/web3.clientVersion.test.js:38]: Should we assert the version ? If the tests fail after the upgrade that's actually ok

File: nodejs/json-rpc-relay/integration.tests/mirror.node/web3.clientVersion.test.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+    test('Should get clientVersion from Arhia Testnet', async () => {
+        // Arrange
+        const web3Provider = new Web3(arkhiaJsonRpcRelayTestnet);
+
+        // Act
+        const result = await web3Provider.eth.getNodeInfo();
+
+        // Assert
+        expect(result).toBeDefined();
```
Comment (daniel-bcw) [inline nodejs/json-rpc-relay/integration.tests/mirror.node/web3.clientVersion.test.js:52]: Should we assert some of the node infos as well, what data are we getting

File: nodejs/json-rpc-relay/integration.tests/mirror.node/web3.clientVersion.test.js [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+    test('Should get clientVersion from Hedera from Arhia Testnet', async () => {
+        // Arrange
+        const ethersProvider = new ethers.providers.JsonRpcProvider(arkhiaJsonRpcRelayTestnet);
+        
+        // Arrange
+        const result = await ethersProvider.send("web3_clientVersion", []);
+
+        // Assert
+        expect(result).toBeDefined();
+        // expect(Number(balance)).toBeGreaterThan(0);
```
Comment (daniel-bcw) [inline nodejs/json-rpc-relay/integration.tests/mirror.node/web3.clientVersion.test.js:67]: Do we need this one? It's from the balance right

File: nodejs/json-rpc-relay/integration.tests/mirror.node/get.hashrate.test.js [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+        "method": "eth_hashrate",
+        "params": [],
+        "id": 0
+      };
+    return data;
+}
+
+describe('[CURL] Hashrate', () => {
+
+    test('Should get 0x0 from Arhia Testnet', async () => {
```
Comment (daniel-bcw) [inline nodejs/json-rpc-relay/integration.tests/mirror.node/get.hashrate.test.js:26]: Curious have we tried to cross check with Hashio? JUst wanna confirm 0x0 is actually the correct hashrate

File: nodejs/json-rpc-relay/integration.tests/mirror.node/eth.syncing.test.js [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+    const data = {
+        "jsonrpc": "2.0",
+        "method": "net_listening",
+        "params": [],
+        "id": 0
+      };
+    return data;
+}
+
+describe('[CURL] Eth Syncing', () => {
```
Comment (daniel-bcw) [inline nodejs/json-rpc-relay/integration.tests/mirror.node/eth.syncing.test.js:25]: Should it be called Eth Signing?

File: nodejs/json-rpc-relay/integration.tests/mirror.node/net.version.test.js (hunk 1 above)
Comment (aaron-bcw) [reply nodejs/json-rpc-relay/integration.tests/mirror.node/net.version.test.js:38]: Yes good test added it

**Pull request merged**